src/etl/NFTHelpers.h

```diff
@@ -0,0 +1,15 @@
+#include <ripple/protocol/STTx.h>
+#include <ripple/protocol/TxMeta.h>
+
+#include <backend/DBHelpers.h>
+
+// Pulling from tx via ReportingETL
+std::pair<std::vector<NFTTransactionsData>, std::optional<NFTsData>>
```

Comment:
Have you considered using a `struct` instead?

Reply:
let's do this refactor later